sources/packages/web/src/assets/css/base.scss

```diff
@@ -55,6 +55,24 @@ strong {
   border-radius: 8px;
 }
 
+.list-color {
```

Comment:
Iam sorry, but why do we need a background for v-list?

Reply:
It's a div with a light theme by default.

Reply:
As per Figma, the background is for the content group (which already the required background color, that we require, I belive), not for the list, and not sure, if why we need a list and row col here
![image](https://github.com/bcgov/SIMS/assets/77353155/d9144771-eb01-464f-9227-ba9a341bbf04)